### PR TITLE
test against Rails 4 and 5

### PR DESCRIPTION
**Pull request by ericcf**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -10,20 +10,13 @@ gemspec
...
-  gem "devise", "~> 3.2"
-  gem "event_capture",
-      tag: "0.1.1",
-      git: "https://github.com/cbitstech/event_capture.git"
-end
-
-group :test do
-  gem "timecop", "~> 0.7"
+# internal CBITs gems
+gem "event_capture", "~> 0.1.2"
```
Comment (michaelwehrley) [inline Gemfile]: 😸

File: spec/dummy/db/schema.rb [hunk 2]
```diff
@@ -68,13 +68,12 @@
     t.inet     "current_sign_in_ip"
     t.inet     "last_sign_in_ip"
     t.boolean  "is_admin"
+    t.index ["email"], name: "index_participants_on_email", unique: true, using: :btree
```
Comment (michaelwehrley) [inline spec/dummy/db/schema.rb]: Nice: `t.index`.

**Comment by michaelwehrley** [comment]
Great work.  Thanks 👍

File: spec/dummy/db/schema.rb (hunk 2 above)
Comment (ericcf) [inline spec/dummy/db/schema.rb]: Yeah, that was added automatically when I migrated locally.

**Pull request merged**